hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java

```diff
@@ -1557,7 +1557,7 @@ public ReplicaHandler recoverAppend(
   @Override // FsDatasetSpi
   public Replica recoverClose(ExtendedBlock b, long newGS,
       long expectedBlockLen) throws IOException {
-    LOG.info("Recover failed close " + b);
+    LOG.info("Recover failed close " + b + ", newGS: " + newGS);
```

Comment:
We can also print `expectedBlockLen` together.

Reply:
@tomscut , Thanks for your review, i will fix it.